### PR TITLE
[Uneven Dream] Chinese(Simplified) update

### PR DESCRIPTION
**Pull request by Sonic714**
23.2.2 uneven dream zh-cn upd

### CODE REVIEW

**Comment by Mimigris** [comment]
screenshot_66
I think you forgot to translate the Origami effect in the _RPG_RT.ldb.po_ file.

Additionally, from what I saw, you changed the title officially used in the Chinese translation of YNOproject from "凹凸梦" to its English title in the introduction and in the loading screen; if your plan is definitively to no longer use the Chinese title, the instructions and the title screen should be edited accordingly.

**Comment by Sonic714** [comment]
> screenshot_66 I think you forgot to translate the Origami effect in the _RPG_RT.ldb.po_ file.
> 
> Additionally, from what I saw, you changed the title officially used in the Chinese translation of YNOproject from "凹凸梦" to its English title in the introduction and in the loading screen; if your plan is definitively to no longer use the Chinese title, the instructions and the title screen should be edited accordingly.

Done as required.

File: unevendream/zh/meta.ini [hunk 1]
```diff
@@ -1,4 +1,4 @@
 [Language]
 Name=简体中文
-Description=Localization by Monkey233 and ynoProjectCN(temporary)
-Term=语言
\ No newline at end of file
+Description=Localization by Moucky233 and ynoProjectCN(temporary)
```
Comment (ynoprojectCN) [inline unevendream/zh/meta.ini:3]:
```suggestion
Description=Localization by Moucky233 and YNOprojectCN translation team (temporary name)
```
Might be a bit more formal this way

**Review by ynoprojectCN — APPROVED** [review]
Checked the files and found only one minor improvement that can be made; the PR is otherwise ready to be merged.

**Pull request merged**